meson.yaml

```diff
@@ -25,9 +25,9 @@ pipeline:
       expected-sha256: 567e533adf255de73a2de35049b99923caf872a455af9ce03e01077e0d384bed
       uri: https://github.com/mesonbuild/meson/releases/download/${{package.version}}/meson-${{package.version}}.tar.gz
 
-  - runs: python3 setup.py build
+  - runs: python3.12 setup.py build
 
-  - runs: python3 setup.py install --prefix=/usr --root=${{targets.destdir}} --skip-build
+  - runs: python3.12 setup.py install --prefix=/usr --root=${{targets.destdir}} --skip-build
```

Comment:
i think if you change this to py/pip-build-install you can just add a 'python3' to environment packages and it will DTRT.